docs/notebooks/flax_basics.ipynb

```diff
@@ -0,0 +1,911 @@
+{
```

Comment:
<p>Nit : Why 101 steps? We usully tart at step 1 with <code>range(1, total_steps + 1)</code></p><p> </p><p>Pystyle : <code>if i % 10 == 0:</code> </p><p> </p><p>Nit (applies to all cells) : Please use single or double quotes consistently.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Tbh because I was lazy and I wanted the 100th step to print ^^ and didn't add a line after the loop. I that's too ugly I can change it ^^. Noted for quotes, fixing</p>